coderd/database/dbtype/dbtype.go

```diff
@@ -1,30 +1 @@
 package dbtype
-
-import (
-	"database/sql/driver"
-	"encoding/json"
```

Comment:
Seems like this is based on your last pr

Reply:
oop